src/components/binding/bindResults.html

```diff
@@ -0,0 +1,41 @@
+<div ng-if="!ctrl.error">
+  <div ng-if="!(ctrl.binding | isBindingReady)" ng-class="{'text-center': !ctrl.progressInline}">
+    <h3 class="bind-pending">
+      <div class="spinner" ng-class="{'spinner-sm': ctrl.progressInline, 'spinner-inline': ctrl.progressInline, 'spinner-xl': !ctrl.progressInline}" aria-hidden="true"></div>
+      <span class="sr-only">Pending</span>
+      <div class="bind-pending-message" ng-class="{'progress-inline': ctrl.progressInline}">The binding was created but is not ready yet.</div>
+    </h3>
+  </div>
+  <div ng-if="(ctrl.binding | isBindingReady)">
+    <h3 class="bind-success">
+      <span class="pficon pficon-ok"></span>
+      <strong>{{ctrl.serviceToBind}}</strong>
+      <span>&nbsp;has been bound</span>
+      <span ng-if="ctrl.applicationToBind">&nbsp;to <strong>{{ctrl.applicationToBind}}</strong> successfully</span>
```

Comment:
there is a known bug here that when you pick the option to just create a secret in the namespace instead of binding to a particular application that this message is wrong, instead of having to cut another common release to pick up a fix to this, can we go ahead and fix it here?